services/web/src/stores/session.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import { merge, omit } from 'lodash';
 import { request } from 'utils/api';
 
+// Stub to enable analytics
+// import { trackSession, setUserId } from 'utils/analytics';
```

Comment:
I would say dont stub just have it call these function and inside utils/analytics have some code that checks to see if GTM is enabled or not

Reply:
That would make it simpler... I'm fine to do that

Reply:
Done